geometry_msgs/PointStamped knock_point
---
bool success
